src/styles/ParentStyles/index.less

```diff
@@ -110,15 +110,18 @@ li {
   text-align: center;
   width: 100%;
 }
+
 .fullButton {
   margin-bottom: 4px;
   width: 100%;
 }
 .events {
-  background-color: #daf4ff;
-  border-radius: 25%;
-  overflow: unset;
-  margin-bottom: 0;
+    background-color: #daf4ff;
+    border-radius: 25%;
+    overflow: unset;
+    margin-bottom: 0;
+    
+
```

Comment:
Couple of extra empty lines to delete here!

Reply:
I will remove the extra spacing, thanks